### PR TITLE
docs: add pr review process

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ZH CARD 2159](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-ciip-portal/2159)

### CODE REVIEW

File: docs/version_control/pull-request-review-process.md
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:14]: Here we could have an example of "what is a good PR Reviews post"

File: docs/version_control/pull-request-review-process.md
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:48]: Pulling a new card into the sprint requires PO approval (taking a card from the sprint backlog doesn't)

File: docs/version_control/pull-request-review-process.md
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:52]: ...Or try to limit the amount of changes per PR, split work where possible ;)

File: docs/version_control/pull-request-review-process.md
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:38]: Documentation about rebasing process before merging changes

**Review by pbastia — CHANGES_REQUESTED** [review]
Thanks for taking that on! Looks great, there are just a few items missing

File: docs/version_control/pull-request-review-process.md
Comment (Sepehr-Sobhani) [reply docs/version_control/pull-request-review-process.md:14]: @pbastia Do you mean something like a screenshot from one of our existing PR review posts?!

Comment (pbastia) [reply docs/version_control/pull-request-review-process.md:14]: Yes, or just a text version of that screenshot, in a frame. something like

> `@PR Reviews` this pull request to do XYZ 
> PR: [link to GH PR](https://github.com/bcgov/cas-home/pull/8) 
> Card: ...

etc

**Review by pbastia — APPROVED** [review]

**Pull request merged**
